earthfile2llb/earthfile2llb.go

```diff
@@ -105,6 +106,11 @@ func Earthfile2LLB(ctx context.Context, target domain.Target, opt ConvertOpt) (m
 		return nil, errors.Wrapf(err, "resolve feature set for version %v for target %s", bc.Earthfile.Version.Args, target.String())
 	}
 
+	err = features.ApplyFlagOverrides(ftrs, os.Getenv("EARTHLY_VERSION_FLAGS"))
```

Comment:
unsure if we should call this `EARTHLY_VERSION_FLAGS` or `EARTHLY_VERSION_FLAG_OVERRIDES`

Reply:
Would this make more sense next to the rest of our config in `main.go`?

Reply:
That's a good point, I can leave it Hidden as it's only (mostly?) a dev feature.

Reply:
100% was assuming it would be hidden.